src/main/java/com/hazelcast/remotecontroller/CloudCluster.java

```diff
@@ -1,13 +1,13 @@
 /**
- * Autogenerated by Thrift Compiler (0.15.0)
+ * Autogenerated by Thrift Compiler (0.13.0)
```

Comment:
I can do that as well. The dependency version should be the same with the compiler used because of the incompatibility between them. Do you prefer that approach?

Reply:
I think it would be better, as I assume these files are generated with 0.15 version for a reason (at least I hope so 😄 )

Reply:
I have done it, though note that still something has changed. Now the server side will use 0.15.0 thrift version instead of 0.13.0